### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by KangZhiDong**
Fix typo

### CODE REVIEW

File: java/org/apache/catalina/core/StandardWrapper.java [hunk 1]
```diff
@@ -1253,7 +1253,7 @@ public synchronized void unload() throws ServletException {
             return;
         unloading = true;
 
-        // Loaf a while if the current instance is allocated
+        // Load a while if the current instance is allocated
```
Comment (gmshake) [inline java/org/apache/catalina/core/StandardWrapper.java]: Not typo here :)

Comment (qixiaobo) [reply java/org/apache/catalina/core/StandardWrapper.java]: Means have a break ?

**Comment by markt-asf** [comment]
It does. The use of loaf is meant to indicate that the thread waits for a while, idle, not doing anything.

**Comment by qixiaobo** [comment]
maybe spin~

**Comment by jfclere** [comment]
I love the definition "to spend time doing nothing, usually when you should be working": macmillandictionary.com

**Comment by qixiaobo** [comment]
> I love the definition "to spend time doing nothing, usually when you should be working": macmillandictionary.com

intelligent

**Pull request closed without merging**